### DISCUSSION TITLE
Compact URL Bar

### DISCUSSION THREAD

**Discussion by Futuremappermydud** [zen-browser/desktop#3183] [Ideas]
Right now the URL Bar looks like this:
![image](https://github.com/user-attachments/assets/89f85f0d-9d5d-4832-aa4b-688835053f29)

While Arc(Windows) looks like this:
![image](https://github.com/user-attachments/assets/4f4a304a-cb6e-4c0b-83ee-1de9b1c57e0e)

The ability to have a shorted url (only the domain) and shrink the url bar to fit the domain would clean up the top of the screen a lot. Since this reduces the clickable space obviously it could be an optional feature/toggle. 

The areas to be trimmed are already highlighted (though deep in firefox source) so it may not be too difficulty to integrate, a hacky fix would need a full build of firefox/zen versus a user available toggle.

**Comment by yusufkandemir** [comment]
Yes, that would be great!

The current URL bar also makes it hard to drag the window since the available space is low. As you said, for some, it would be preferable to have it take up more space to have an increased click area for the URL bar. So, it's better to make it optional.
A current workaround to make it smaller is adding more "Flexible Space" to both sides of the URL bar from the "Customize Toolbar" option, which can be found by right-clicking the toolbar.

As for having only the domain, I would not use it myself, but I can see how it can be less distracting and visually appealing. So, just like in Arc, it can be added as an optional feature.
